application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/excel/SpreadSheetImportTest.java

```diff
@@ -54,7 +54,7 @@
         TEST_CASE_TYPE;
     private static final String GET_CASE_TYPES_COUNT_QUERY = "SELECT COUNT(*) FROM case_type";
 
-    private static final String RESPONSE_JSON = "GetCaseTypesResponseForCCD_TestDefinition_V35.json";
```

Comment:
Why a jump from V35 to V43?

Reply:
I just made it to be in sync with the excel file as both need to go hand in hand

Reply:
ok makes sense